README.md

```diff
@@ -29,7 +29,8 @@ Use the following pattern to link revision data to Gerrit:
 |--------------------|---------------------|
 | v <= 3.5.x         | v <= 1.3.2          |
 | 3.5.x < v <= 4.0.0 | 2.0.0 <= v <= 2.0.1 |
-| 4.0.1 <= v         | 2.0.2 <= v          |
+| 4.0.1 <= v < 4.1.0 | v2.0.2              |
```

Comment:
So there cannot be any more updates for GE4.0?
There is at least #63...

Reply:
Hi @gerhardol, regarding #63 - I can't reproduce it on my side yet and there is an unknown Gerrit plugin version author used to get that error.